packages/sdk/src/impl/VCLImpl.ts

```diff
@@ -54,35 +65,33 @@ export class VCLImpl implements VCL {
     credentialTypes: Nullish<VCLCredentialTypes>;
     credentialTypesModel: Nullish<CredentialTypesModel>;
     credentialTypeSchemas: Nullish<VCLCredentialTypeSchemas>;
-    verifiedProfileUseCase = VclBlocksProvider.provideVerifiedProfileUseCase();
-    jwtServiceUseCase = VclBlocksProvider.provideJwtServiceUseCase();
-    profileServiceTypeVerifier = new ProfileServiceTypeVerifier(
-        this.verifiedProfileUseCase
-    );
 
-    identificationUseCase = VclBlocksProvider.provideIdentificationUseCase();
```

Comment:
There should be only non-nullable model data structures:
    credentialTypesModel: CredentialTypesModel
    credentialTypeSchemasModel: CredentialTypeSchemasModel
    countriesModel: CountriesModel
Please remove:
    countries: Nullish<VCLCountries>;
    credentialTypes: Nullish<VCLCredentialTypes>;
    credentialTypeSchemas: Nullish<VCLCredentialTypeSchemas>;